README.md

```diff
@@ -3,4 +3,8 @@ A SAT solver written in Nim.
 
 ```nim
 import sat/[sat, satvars]
-```
\ No newline at end of file
+```
+
+Note:
```

Comment:
Note: We really need to fix this one too.

Reply:
Yah, seems like a gotcha.